src/diffusers/pipelines/audioldm2/pipeline_audioldm2.py

```diff
@@ -0,0 +1,962 @@
+# Copyright 2023 The HuggingFace Team. All rights reserved.
```

Comment:
AudioLDM authors too?

Reply:
Reolved in https://github.com/huggingface/diffusers/pull/4549/commits/fc96707d54ef33c8ef471b393f139e01de4fc567